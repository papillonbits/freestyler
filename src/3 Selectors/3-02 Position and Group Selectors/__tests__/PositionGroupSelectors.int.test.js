import { regular } from './PositionGroupSelectors.int.story'

describe('<PositionGroupSelectors />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
