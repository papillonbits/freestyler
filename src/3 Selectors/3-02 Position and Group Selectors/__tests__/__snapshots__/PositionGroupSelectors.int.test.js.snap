// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PositionGroupSelectors /> Render must match regular() 1`] = `
<div>
  <div
    class="position-group-selectors"
  >
    <h1>
      Position and Group Selectors
    </h1>
    <p
      class="my-class"
    >
      p.my-class
    </p>
    <div
      id="my-id"
    >
      <ol>
        <li>
          div ol li
        </li>
        <li>
          div ol li
        </li>
        <li>
          <p
            class="my-class"
          >
            div ol li p.my-class
          </p>
        </li>
      </ol>
    </div>
  </div>
</div>
`;
